src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -541,6 +545,23 @@ pub struct Args {
     // === Tracing options. ===
     #[clap(flatten)]
     tracing: TracingCliArgs,
+
+    // === Testing options. ===
+    /// Injects arbitrary whitespace into builtin table fingerprints, which can
+    /// trigger builtin item migrations. The amount of whitespace is determined
+    /// by
+    /// `really_dangerous_do_not_call_this_in_production_table_fingerprint_whitespace_version`.
+    /// This argument is meant for testing only and as the names suggests
+    /// should not be set in production.
+    #[clap(long, arg_enum, requires = "unsafe-mode")]
+    unsafe_builtin_table_fingerprint_whitespace: Option<UnsafeBuiltinTableFingerprintWhitespace>,
+    /// Controls the amount of whitespace injected by
+    /// `really_dangerous_do_not_call_this_in_production_table_fingerprint_whitespace`.
+    /// Incrementing this value can allow triggering multiple builtin
+    /// migrations from a single test. This argument is meant for testing only
+    /// and as the names suggests should not be set in production.
+    #[clap(long, requires = "unsafe-mode", default_value = "1")]
+    unsafe_builtin_table_fingerprint_whitespace_version: usize,
```

Comment:
This is a bit unergonomic for tests, we have to override and increase the size for each restart in order to stress test, right?

Reply:
Actually I don't see where this variable is used. We appear to only add one "\n" in each case

Reply:
> This is a bit unergonomic for tests, we have to override and increase the size for each restart in order to stress test, right?

Yeah if you want to test multiple builtin item migrations in the same test, then you'll have to increment this for each restart. Is there something that would be more ergonomic?

> Actually I don't see where this variable is used. We appear to only add one "\n" in each case

It's used right below this on line 606

```Rust
let whitespace = "\n".repeat(args.unsafe_builtin_table_fingerprint_whitespace_version);
```

Reply:
Another option I considered was to omit `unsafe_builtin_table_fingerprint_whitespace_version` and only allow a single migration per test.